src/meta/src/manager/cluster.rs

```diff
@@ -492,6 +493,33 @@ impl ClusterManager {
     pub async fn get_worker_by_id(&self, worker_id: WorkerId) -> Option<Worker> {
         self.core.read().await.get_worker_by_id(worker_id)
     }
+
+    pub async fn get_back_pressure(&self) -> MetaResult<GetBackPressureResponse> {
+        let mut core = self.core.write().await;
+        let mut back_pressure_infos: Vec<BackPressureInfo> = Vec::new();
+        for worker in core.workers.values_mut() {
+            if worker.worker_type() != WorkerType::ComputeNode {
```

Comment:
Should also check whether it's streaming computer node.

Reply:
I tought this is enough to check whether it is a compute node.
`worker.worker_type() != WorkerType::ComputeNode`

Do i need performance more check?